NEWS.md

```diff
@@ -1,3 +1,41 @@
+### v8.11.0 (2022-05-11)
+
+* Log forwarding from APM agent is now possible.
+
+    * Added application log aggregator to collect logs with adaptive sampling and appropriate max samples stored.
+
+    * Added `application_logging` configuration and env vars with appropriate defaults.
+
+    * Changed default value of `application_logging.enabled` to be true.
+
+    * Set `application_logging.forwarding.enabled` to false when HSM is set.
```

Comment:
i'm not sure people know what HSM is, maybe spell out high security mode

Reply:
Since we might use HSM elsewhere, go with "High Security Mode (HSM)" on any first occurrence.